server/controllers/armodels.controller.js

```diff
@@ -12,6 +12,16 @@ exports.getModels = async ctx => {
   }
 };
 
+exports.getModelById = async ctx => {
+  try {
+    ctx.body = await ArModels.findById(ctx.params.id);
+    ctx.status = 200;
+  } catch (error) {
```

Comment:
As I mentioned, not doing anything with the error catched, maybe log it or throw a new Error(error)?